test_setup/cy_interface/interface.json

```diff
@@ -97,7 +97,9 @@
                            "[data-cy=\"user options admin 5cfb8cb4-26db-4204-a4d0-7c822eda4291 0eab74c9-f13b-4343-ab71-5e50875f8403\"]",
                            "[data-cy=\"user options alice_smith 5cfb8cb4-26db-4204-a4d0-7c822eda4291 0eab74c9-f13b-4343-ab71-5e50875f8403\"]"
                         ],
-                        "home": []
+                        "home": [
+                           "[data-cy=\"connectObject options 696cfe80-8a3e-477f-a566-3c14dac1cc2d ea33376e-a991-497f-93e1-8d853cc27de2 0eab74c9-f13b-4343-ab71-5e50875f8403\"]"
+                        ]
```

Comment:
I'm curious what your using to generate the data-cy here. What are the 3 ids? Could it be shorter?